etc/kayobe/environments/aufn-ceph/a-universe-from-nothing.sh

```diff
@@ -10,8 +10,8 @@
 set -eu
 
 BASE_PATH=~
-KAYOBE_BRANCH=stackhpc/yoga
-KAYOBE_CONFIG_BRANCH=stackhpc/yoga
+KAYOBE_BRANCH=stackhpc/2023.1
+KAYOBE_CONFIG_BRANCH=stackhpc/2023.1
 KAYOBE_ENVIRONMENT=aufn-ceph
 
 PELICAN_HOST="10.0.0.34 pelican pelican.service.compute.sms-lab.cloud"
```

Comment:
There are more changes to make in this file.
We're moving away from ``virtualenv`` so the ``python3-virtualenv`` package needs to go.
so does ``virtualenv kayobe``
We should check the images we use (we're moving to the ark-hosted images for aufn in the future) and make sure we're not missing any packages.
Ideally we should be testing these changes before making them. With the 'wrong' branch at the top, it's clear this script is outdated. If we haven't tested it, I think we should have a big clear warning at the top of the file